### DISCUSSION TITLE
how to install v0.6 / improve doc

### DISCUSSION THREAD

**Discussion by renarena** [BeardedTek-com/fEVR#88] [Q&A]
Another time i try to install fevr, so i list what was unclear for me / could be changed to help others with installation.
And it does not work, what is wrong?

I miss some things in https://fevr.video/readme/ .  Especially please add, in which location/file code blocks should be put.

> Requirements
MQTT Broker (if you have frigate running, you have this) listening to 0.0.0.0
    Example mosquitto.conf listener section if using port 1883
listener 1883 0.0.0.0

I have HAOS in a VM and another VM with fevr and frigate using mqtt from HA. No idea how to put this line using the HA addon, but i can access mqtt from frigate.

I cloned the git repo, but was not sure if its still needed. If yes, add the command line to do this?

There is some code for .env and another box, but without mention, this has to go into docker-compose.yml and no sign, the next line is a terminal command (i know this is very basic, but not everybody is experienced with git and docker.

> Configure Frigate
   make one entry called 'frigate' (without the quotes) with your internally accessible frigate URL

Without reading the doc i would not know, what to put in. The field in settings says "Frigate MQTT name" probably i need the MQTT topic used by frigate. Url of frigate .Or of Mqtt? I used frigates url.

I generated the mqtt API key 

> Configure mqtt_client
> Once you have your API Key you can generate your config file for the mqtt_client.
   [See mqtt_client docs here](https://fevr.video/readme/docs/MQTT_CLIENT.md) for an example

The link is wrong, i had to lookup the mqtt docs file in git.

Again there are several code blocks but no mention into which file they belong. Probably config.yml? But this is an folder. If i remove it i get an docker error on restart.
So i put the file as config.yml inside config.yml. 
Do i need the api key in any case or only if i do not provide user/password?

I dont see any events on fevr all day, 

In settings/frigate i have 2 entries, a wrong one with mqtt of HA. There is no button do delete and if i empty the fields and press edit the page get an 404 error and there are errors in the docker log:
```
sqlalchemy.exc.IntegrityError: (sqlite3.IntegrityError) UNIQUE constraint failed: frigate.name
[SQL: UPDATE frigate SET url=?, name=? WHERE frigate.id = ?]
[parameters: ('', '', 1)]
```
I have no idea how to delete wrong entries.

Do i press edit before i want to change or after? Its not self explanatory and not consistent in several different software...

Maybe i am old fashioned, but i dont like dialogs with no save button. How can i go back without saving after trying something?

**Comment by BeardedTek** [comment]
Thank you for the input.  This is definitely one area I struggle at is documentation.  I'm working on it.

I'll have to comb through the docs the next couple days and make some edits.

This Friday I have some time carved out to work on things.  If you have some time, please drop into the [telegram support channel](https://t.me/BeardedTekfEVR) or message me directly on telegram, or matrix @Beardedtek and I'll walk you through things as I can in a more real time format.

**Reply by renarena** [reply]
which matrix server?

**Comment by renarena** [comment]
how can i contact you in matrix? i tried @BeardedTek on matrix.org

**Reply by BeardedTek** [reply]
I just created a fEVR Support Space to make it easier.  I'm available from 9:30am - 1:00PM ADST(GMT-8) M-F

https://matrix.to/#/#fevrsupport:matrix.org

**Reply by renarena** [reply]
I am not allowed to write.

**Reply by BeardedTek** [reply]
What is your matrix username?  I'll send you an invite.  All channels inside the space should be publicly accessible.
